libraries/src/Form/Field/FilelistField.php

```diff
@@ -187,17 +187,21 @@ public function setup(\SimpleXMLElement $element, $value, $group = null)
 	 */
 	protected function getOptions()
 	{
-		$options = array();
+		$options = [];
+		$path    = $this->directory;
 
-		$path = $this->directory;
-
-		if (!is_dir($path))
+		if (strpos($path, '/') === 0)
```

Comment:
```suggestion
		if (strpos($path, '/') !== 0)
```
We want to prepend JPATH_ROOT if the path does NOT start with a slash. This is the case if strpos return either false (no slash in the path at all) or something greater than zero (some shalsh but not at the beginning.